### PR TITLE
Push hook

### PR DESCRIPTION
**Pull request by geenen124**
Add git hooks folder and installation functionality. See the updated README.md for instructions

### CODE REVIEW

**Comment by bartdejonge1996** [comment]
There's a spelling error present in the Readme (necessary for this project run), can you fix this quickly?

File: README.md [hunk 1]
```diff
@@ -2,4 +2,10 @@ This is the Context project of the team goto fail;
 
 
 To use lombok on the project, you need to enable annotation processing in your settings.
-In IntelliJ, this can be done in your settings under: "Build, Execution, Deployment > Compiler > Annotation Processors". You only need to click the checkbox at the top of the page.
\ No newline at end of file
+In IntelliJ, this can be done in your settings under: "Build, Execution, Deployment > Compiler > Annotation Processors". You only need to click the checkbox at the top of the page.
+
+To add git hooks necessary for this project run:
```
Comment (bartdejonge1996) [inline README.md]: Spelling error

**Comment by bart-de-jonge** [comment]
Great work, it's always nice to test offline to prevent lots of build errors on travis!

**Comment by mennoo1996** [comment]
Nice work!

**Pull request merged**